### PR TITLE
Implemented ShakerSort class

### PR DESCRIPTION
**Pull request by Ri0ee**

### CODE REVIEW

File: sorting/src/dip107/ShakerSort.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+public class ShakerSort {
+    private static void swap(int[] data, int a, int b) {
+        int temp = data[a];
+        data[a] = data[b];
+        data[b] = temp;
+    }
+
+    public static void sort(int[] data) {
+        int l = 0;
```
Comment (cyoq) [inline sorting/src/dip107/ShakerSort.java]: l -> left

File: sorting/src/dip107/ShakerSort.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        int r = data.length - 1;
```
Comment (cyoq) [inline sorting/src/dip107/ShakerSort.java]: r -> right

**Review by cyoq — CHANGES_REQUESTED** [review]
Change names to variables

File: sorting/src/dip107/ShakerSort.java (hunk 1 above)
Comment (Ri0ee) [reply sorting/src/dip107/ShakerSort.java]: every sane programmer knows the meaning of "l" and "r", I feel no need in such changes!

File: sorting/src/dip107/ShakerSort.java (hunk 2 above)
Comment (Ri0ee) [reply sorting/src/dip107/ShakerSort.java]: every sane programmer knows the meaning of "l" and "r", I feel no need in such changes!

File: sorting/src/dip107/ShakerSort.java (hunk 1 above)
Comment (cyoq) [reply sorting/src/dip107/ShakerSort.java]: I am not a sane programmer, so I do not know, what l and r mean

**Review by cyoq — APPROVED** [review]

**Pull request merged**